.github/workflows/unit-tests.yml

```diff
@@ -32,6 +32,7 @@ jobs:
             "common-1",
             "common-2",
             "common-3",
+            "xmodule-1"
```

Comment:
👍🏻 

This isn't something you need to fix now, but it's unfortunate that we need to list shards both here and in `unit-test-shards.json`. It would be really easy to add a new shard in the JSON file but forget to add it here. Could you ask your tech lead to add this to your backlog?

Reply:
Yeah sure... will create a ticket for this.

Reply:
Fair enough.